modules/claims_api/lib/claims_api/claim_logger.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  # Logger for ClaimsApi, mostly used to format a print statement
+  # Uses the standard Rails logger
+  #
+  #   ClaimsApi::Logger.log('526ez', claim_id: 'id_here')
+  #   # => "ClaimsApi :: 526ez :: Claim ID: id_here"
+  class Logger
+    LEVELS = %i[debug info warn error fatal unknown].freeze
+
+    def self.log(tag, **params)
+      level = pick_level(**params)
+      params.delete(:level)
+      msg = format_msg(tag, **params)
+
+      Rails.logger.send(level, msg)
+      msg
+    end
+
+    def self.pick_level(**params)
+      params.key?(:level) && params[:level].to_sym.in?(LEVELS) ? params[:level].to_sym : :info
+    end
+
+    def self.format_msg(tag, **params)
+      msg = ['ClaimsApi', tag]
+      case tag
+      when '526ez'
```

Comment:
Should consider making the tag values into constants.
So we can call the logger like :: `ClaimsApi::Logger.log(ClaimsApi::Logger::526, claim_id: 'id_here')`

Otherwise, we're gonna run into situations where we try to call the logger like :: `ClaimsApi::Logger.log('526', claim_id: 'id_here')` which won't find a match in the `case` and lead to a useless log message.

Reply:
Or add a method that validates that the value given for `tag` is a recognized value.